app/helpers/patient_details_helper.rb

```diff
@@ -173,6 +173,7 @@ def custom_export_details
       age: calc_current_age || '',
       jurisdiction_name: jurisdiction[:name] || '',
       symptom_onset_defined_by: user_defined_symptom_onset ? 'User' : 'System',
+      no_symptom_history: symptom_onset.nil?.to_s,
```

Comment:
Why the change to a string?

Reply:
don't remember why I did this by I removed it and the tests still pass, so it's probably not necessary